M/MillenniumDB/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MillenniumDB"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/MillenniumDB/MillenniumDB.git", "b1df3251fa1e51ad160cae1e0a57553da93724be"),  # TODO: use main branch, this is a temporary fix
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/MillenniumDB
+
+if [[ "${target}" == x86_64-* ]] || [[ "${target}" == i686-* ]] || [[ "${target}" == armv7l-* ]] || [[ "${target}" == powerpc64le-* ]] || [[ "${target}" == armv6l-* ]]; then
```

Comment:
```suggestion
if [[ "${target}" == x86_64-* ]] || [[ "${target}" == i686-* ]] || [[ "${target}" == arm-* ]] || [[ "${target}" == powerpc64le-* ]]; then
```
although I don't understand why don't you do
```sh
if [[ "${target}" == aarch64-* ]]; then
    # ...
else
    # ...
fi
```
:slightly_smiling_face: